### ISSUE TITLE
Unexpected token P in JSON at position 1

### ISSUE THREAD

**Issue by SlashKirill** [anvilco/spectaql#83]
Hi,

i'm getting following error with almost standard config and SDL from latest postgraphile:

```
bash-3.2$ npx spectaql config.yml 
Unexpected token P in JSON at position 1
```

sdl: [sdl.txt](https://github.com/anvilco/spectaql/files/6943358/sdl.txt)

config.yml

```
spectaql:
#  logoFile: ./test/fixtures/logo.png
#  faviconFile: ./test/fixtures/favicon.png

introspection:
  schemaFile: ./sdl.gql
#  metadataFile: ./examples/data/metadata.json
#  dynamicExamplesProcessingModule: ./examples/customizations/examples

info:
  title: GraphQL API Reference
  description: Welcome to the party!
  termsOfService: https://www.example.com/terms
  contact:
    name: API Support
    url: http://www.example.com/support
    email: support@example.com
  license:
    name: Apache 2.0
    url: https://www.apache.org/licenses/LICENSE-2.0.html

servers:
  - url: https://example.com/graphql
    description: Production
    production: true
```

Any suggestions what is the problem? Thanks.

**Comment by newhouse** [comment]
It seems like it's a problem in the `graphql-2-json-schema` dependency...which I am also a contributor/maintainer of. It's not handling argument defaults that are ENUM lists properly.  Could you open a ticket [over there](https://github.com/charlypoly/graphql-to-json-schema) and I'll have a look?

FYI, the stack trace I'm seeing is this:
```sh
undefined:1
[PRIMARY_KEY_ASC]
 ^

SyntaxError: Unexpected token P in JSON at position 1
    at JSON.parse (<anonymous>:null:null)
    at Object.resolveDefaultValue (/path/to/spectaql/node_modules/graphql-2-json-schema/dist/lib/reducer.js:51:16)
    at introspectionFieldReducer (/path/to/spectaql/node_modules/graphql-2-json-schema/dist/lib/reducer.js:38:46)
    at arrayReduce (/path/to/spectaql/node_modules/lodash/lodash.js:697:21)
    at Function.reduce (/path/to/spectaql/node_modules/lodash/lodash.js:9749:14)
    at introspectionFieldReducer (/path/to/spectaql/node_modules/graphql-2-json-schema/dist/lib/reducer.js:26:46)
    at arrayReduce (/path/to/spectaql/node_modules/lodash/lodash.js:697:21)
    at Function.reduce (/path/to/spectaql/node_modules/lodash/lodash.js:9749:14)
    at /path/to/spectaql/node_modules/graphql-2-json-schema/dist/lib/reducer.js:59:34
    at arrayReduce (/path/to/spectaql/node_modules/lodash/lodash.js:697:21)
    at Function.reduce (/path/to/spectaql/node_modules/lodash/lodash.js:9749:14)
    at Object.fromIntrospectionQuery (/path/to/spectaql/node_modules/graphql-2-json-schema/dist/lib/fromIntrospectionQuery.js:30:30)
    at jsonSchemaFromIntrospectionResponse (/path/to/spectaql/app/spectaql/graphql-loaders.js:88:20)
    at module.exports (/path/to/spectaql/app/spectaql/index.js:108:22)
    at loadData (/path/to/spectaql/app/index.js:169:74)
    at module.exports (/path/to/spectaql/app/index.js:173:80)
    at Object.<anonymous> (/path/to/spectaql/bin/spectaql.js:70:1)
    at Module._compile (internal/modules/cjs/loader.js:1076:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:941:32)
    at Function.Module._load (internal/modules/cjs/loader.js:782:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
```

**Comment by SlashKirill** [comment]
> It seems like it's a problem in the `graphql-2-json-schema` dependency...which I am also a contributor/maintainer of. It's not handling argument defaults that are ENUM lists properly. Could you open a ticket [over there](https://github.com/charlypoly/graphql-to-json-schema) and I'll have a look?

Just in case, I opened a ticket there.

**Comment by newhouse** [comment]
Fix coming...

**Comment by newhouse** [comment]
This is fixed in `v0.7.2`
